WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -134,10 +136,22 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetScreen:(FBRouteRequest *)request
 {
-  FBSession *session = request.session;
-  CGSize statusBarSize = [FBScreen statusBarSizeForApplication:session.activeApplication];
+  XCUIApplication *app = XCUIApplication.fb_systemApplication;
+
+  CGSize statusBarSize = app.statusBars.allElementsBoundByIndex.firstObject.frame.size;
```

Comment:
Well, it returned just zero, zero as the TV output example in the PR’s description. It can return the CGSizeZero like current master’s implementation. Let me double check later

Reply:
ah, amounts. I'm not sure if such a case exists (tvOS and iPhone I tested did not have), but let me add the case